src/controllers/api/clients/bookings.js

```diff
@@ -0,0 +1,67 @@
+import models from '../../../database/models';
+import { getToken } from '../../create_token'
+
+export const getBookings = (req, res) => {
+  models.sequelize.query("SELECT ext_bookings.event_title, bookings.start_at, bookings.end_at,bookings.room_id, events_courses.title FROM ext_bookings RIGHT OUTER JOIN bookings ON bookings.id = ext_bookings.booking_id LEFT OUTER JOIN events_courses ON events_courses.id = bookings.event_id").spread((results, metadata) => {
```

Comment:
use ORM

Reply:
I couldn't make it with ORM, as I searched there is no right outer join in ORM
If there is a way, I would be grateful if you give me a hint